src/Ticket.java

```diff
@@ -58,5 +85,43 @@ public String toString() {
                 ", price=" + price +
                 '}';
     }
+
+    @Override
+    public String toString() {
+        return getAllValues();
```

Comment:
So what the benefit of having toString and getAllValues together?
I see no any reason)